src/main.tsx

```diff
@@ -1,10 +1,11 @@
-import './index.scss';
+import './main.scss';
+import '@fontsource/roboto';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom/client';
-import { App } from './App';
+import { Footer } from './templates/Footer/Footer';
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
   <React.StrictMode>
-    <App />
```

Comment:
we should keep `App.tsx`

This is also a place where you should use  `<ThemeProvider>` to inject our custom theme

Reply:
Should be fixed in 2b03974